Add self-service registration (#331)
DFE-Digital/bat-design-history: app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

---

> title: Self-service registration for accrediting bodies and their training providers

Reviewer: ```suggestion
title: Let users at accredited bodies set up an account for themselves and users at their partner training providers
```

Author: We refer to "higher education institutions" rather than "accredited bodies" since the latter could also include SCITTS, which aren't part of this work. We've also used "school direct partners" in relation to training providers to be more explicit.

I will combine your suggestion with the preferred nomenclature.

> description: Let higher education institutions register on Manage and invite their school direct partners

Reviewer: ```suggestion
description: Exploring ways to help accredited body users invite users at their partner training providers to create accounts to use the Manage service
```

Author: We refer to "higher education institutions" rather than "accredited bodies" since the latter could also include SCITTS, which aren't part of this work. We've also used "school direct partners" in relation to training providers to be more explicit.

I will combine your suggestion with the preferred nomenclature.

date: 2021-03-18
---

{% from "figure/macro.njk" import appFigure with context %}

> The aim of the work was to answer the following question:

Reviewer: Kill

Author: Done

---

> title: Self-service registration for higher education institutions and their school direct partners

Reviewer: Can we avoid ‘self service’ as a term?

Author: Can you recommend an alternative to 'self-service'?

---
title: Self-service registration for higher education institutions and their school direct partners
description: Exploring ways to help higher education institutions register and invite their school direct partners to use the Manage service
date: 2021-03-18
---

{% from "figure/macro.njk" import appFigure with context %}

> How might we provide a self-service journey for higher education institutions (HEIs) and their school direct partners (SDs) to register on the Manage service?

Reviewer: Change to a normal intro - not a question.

Author: I have updated as per your suggestion

## Why we are doing this

Around 70 HEIs have not yet registered to use Manage. On average each of them accredits courses for 10 SDs, many of which are also not registered on Manage. We do not have contact details for all the SDs.

We need to make sure that all these organisations have registered to use Manage before the UCAS service is turned off in October 2021.

> Adding them all manually will be a great deal of work for the transition team. We want to make the process more efficient, while not placing a significant burden on HEIs or SDs.

Reviewer: ```suggestion
Adding them all manually will be a lot of work for the transition team. We want to make the process more efficient, while not placing a significant burden on HEIs or SDs.
```

## Requirements

Before we started designing possible solutions, we spoke to members of the provendor and transition teams about their requirements.

> These requirements are not presented as user needs, since this work is being done primarily for the benefit of teams within DfE.

Reviewer: I wonder if these needs pointing out?

Author: I think it's important to note where the requirements came from, i.e., not from a validated user need.

What do you think @johndoates?

### Getting contact details

We need to make sure we have an up to date contact name and email address for each SD.

### Inviting training providers

We need to:

- reduce the amount of manual work done by the transition team, to save team effort and reduce the risk of errors
- know which HEIs are on Manage
- know which SDs have been set up for each HEI
- know which SDs have set up and opened courses on Manage

## Provider data analysis

There are 87 HEIs in the Manage database. 22 HEIs have been set up to use Manage and the remaining 65 have not.

### Number of SDs connected to each HEI

Using the Publish database, we looked at how many SDs are connected to each HEI.

- 25% of HEIs have no SDs
- 24% of HEIs have between 1 and 10 SDs
- 36% of HEIs have between 11 and 20 SDs
- 15% of HEIs have more than 20 SDs
- Average number of SDs per HEI: 10
- Maximum number of SDs per HEI: 33
- Minimum number of SDs per HEI: 0

If we ask an HEI user separately about each SD then on average they will see 10 pages of questions.

### Number of HEIs connected to each SD

- 77% of SDs have only 1 HEI
- 14% have 2 HEIs
- 6% have 3 HEIs
- 3% have more than 3 HEIs

> This is important since if an SD has multiple HEIs, they only need to be onboarded once. The other HEIs accrediting their courses will not need to provide contact details for them.

Reviewer: ```suggestion
This is important because if an SD has multiple HEIs, they only need to be registered once. The other HEIs ratifying their courses will not need to provide contact details for them.
```

### Cross-over between users of Manage and Publish

#### Manage users in publish

- 59% of Manage users are in Publish
- 68% of active Manage users are in Publish
- 63% of active Manage users are active in Publish

#### Publish users in Manage

- 20% of Publish users are also in Manage
- 27% of active Publish users are in Manage
- 23% of active Publish users are active in Manage

> Since there are a lot of users in Publish who are not set up in Manage, we have a large pool of contact details to draw from. These users will already have DfE Sign In accounts.

Reviewer: ```suggestion
As there are a lot of Publish users who are not on Manage, we can use their information to ease the registration process as they’ll already have DfE Sign-in accounts.
```

## Iteration 1

> We decided to start by prototyping a small service which would allow HEI users to provide us with the information we need.

Reviewer: Is it a separate service or a flow?

Author: It is a separate flow. I have amended the wording.

### What we did and why

In our first iteration, we wanted HEI users to answer two questions.

- Do you want to open courses at the training provider?
- Who do you want to manage teacher training applications at the provider?

This would allow us to invite the right people at each SD, while leaving the HEI in control of when their courses become available.

{{ appFigure({
  image: {
    file: "self-service-registration-v1-flow.png",
    alt: "Iteration 1 user flow"
  },
  caption: "Figure 1: Iteration 1 user flow"
}) }}

The journey began on a ‘start page’ listing the SDs working with the HEI and setting out the steps involved in creating an account.

The aim was to give the user a chance to gather information before starting, which is important since it is not possible to save and return to the process.

> Users would reach the start page from an email. For this iteration we did not include DfE Sign In account creation, as we assumed that the email would contain a ‘magic link’ bypassing it.

Reviewer: Change DfE Sign In to DfE Sign-in everywhere

Author: I have updated as suggested.

{{ appFigure({
  image: {
    file: "self-service-registration-v1-01.png",
    alt: "Start page asking user to set up their account"
  },
  caption: "Figure 2: Set up your account"
}) }}

> In this iteration we assumed that HEIs may not want to make courses available on Find straight away.

Reviewer: ```suggestion
In this iteration we assumed that HEIs may not want to open courses on Find during this process.
```

After the start page, the user was asked whether they would like to open courses at each SD. For this iteration we set this out as one SD per page.

If the user said they would like to open courses then they were asked for contact details. No fields were pre-filled as we assumed we wanted to gather fresh information.

{{ appFigure({
  image: {
    file: "self-service-registration-v1-02.png",
    alt: "Question page asking ‘do you want to open courses at the school direct?’"
  },
  caption: "Figure 3: Question page asking ‘Do you want to open courses at the school direct?’"
}) }}

Once the user had decided whether or not to open courses for each SD, they were asked to complete a data sharing agreement (DSA).

We took the existing DSA directly from Manage without any changes. Our main concern was not readability, but whether users felt they could sign it or would need to refer to someone else.

{{ appFigure({
  image: {
    file: "self-service-registration-v1-03.png",
    alt: "Data sharing agreement"
  },
  caption: "Figure 4: Data sharing agreement"
}) }}

> On the check answers page we needed to make it clear that clicking the green button would send invitations to SDs. We also decided to remind the user that they would be agreeing to the DSA.

Reviewer: ```suggestion
On the ‘check answers’ page we needed to make it clear that clicking ‘send invites’ would send invitations to SDs. We also decided to remind the user that they would be agreeing to the DSA.
```

{{ appFigure({
  image: {
    file: "self-service-registration-v1-04.png",
    alt: "Check your answers page"
  },
  caption: "Figure 5: Check your answers page"
}) }}

We used a standard success page and did not tailor it to this journey at all. This was largely due to a lack of time, but also because we did not know what the next steps would be.

{{ appFigure({
  image: {
    file: "self-service-registration-v1-05.png",
    alt: "Success page"
  },
  caption: "Figure 6: Success page"
}) }}

> ## Iteration 2: using the task list pattern

Reviewer: Can you add a little comment to say why we discarded this approach?

For this version, we explored using the task list pattern to allow a non-linear journey. Users could use the task list to choose the order in which to provide contact details.

{{ appFigure({
  image: {
    file: "self-service-registration-v2-flow.png",
    alt: "Iteration 2 user flow"
  },
  caption: "Figure 7: Iteration 2 user flow"
}) }}

The task list page gave two tasks for each SD:

- decide whether to make courses available
- decide who to invite

The user could only complete the second task once the first had been completed.

{{ appFigure({
  image: {
    file: "self-service-registration-v2-02.png",
    alt: "The task list"
  },
  caption: "Figure 8: The task list"
}) }}

## Iteration 3: assuming courses will be opened

> We created a third iteration to test with users.

Reviewer: Kill?

Author: Removed as suggested.

In this iteration we assumed that all courses would automatically be opened once the relevant HEI and SD were registered on Manage.

We removed the ‘Do you want to open courses at the training provider?’ question and went straight to asking for contact details.

> We did not use the task list pattern as we decided it added unnecessary complications.

Reviewer: Elaborate on one or two unnecessary complications

Author: Elaborated as suggested.

}) }}

{{ appFigure({
  image: {
    file: "self-service-registration-v3-02b.png",
    alt: "Question page asking ‘Who do you want to invite to manage applications at the training provider?’ and showing the ‘someone else’ option fields"
  },
  caption: "Figure 12: Question page asking ‘Who do you want to invite to manage applications at the training provider?’ and showing the ‘someone else’ option fields"
}) }}

{{ appFigure({
  image: {
    file: "self-service-registration-v3-02c.png",
    alt: "Question page asking ‘Who do you want to invite to manage applications at the training provider?’ only showing name and email fields as there are no known people in the organisation"
  },
  caption: "Figure 13: Question page asking ‘Who do you want to invite to manage applications at the training provider?’ only showing name and email fields as there are no known people in the organisation"
}) }}

The data sharing agreement remained the same as in Manage.

{{ appFigure({
  image: {
    file: "self-service-registration-v3-03.png",
    alt: "Data sharing agreement"
  },
  caption: "Figure 14: Data sharing agreement"
}) }}

> We removed mention of the DSA from the ‘check your answers’ page since there was already a confirmation check box at the end of the DSA itself.

Reviewer: sometimes the post says ‘check your answers’ and sometimes check answers without quotes.

I would suggest: `‘check answers’ page` throughout.

Author: I have updated as suggested.
